extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -428,4 +434,18 @@ private function getTrendingVideosData() {
 
 		return $data;
 	}
+
+	private function getWikiStats() {
+		global $wgCityId;
+
+		$wikiStats = [];
+		$service = new WikiDetailsService();
+		$wikiDetails = $service->getWikiDetails( $wgCityId );
+
+		if ( !empty( $wikiDetails['stats'] ) ) {
```

Comment:
As we always want to return some value or empty value is that `if` even nessesary? Wouldn't be better to replace that block with just `return $wikiDetails['stats']`?